save-core/src/commonMain/kotlin/org/cqfn/save/core/utils/ProcessBuilder.kt

```diff
@@ -100,6 +106,17 @@ class ProcessBuilder {
         } ?: logDebug("Execution output:\n$stdout")
         return ExecutionResult(status, stdout, stderr)
     }
+
+    /**
+     * Log error message and exit with status = -1
+     *
+     * @param errMsg error message
+     * @return [ExecutionResult] corresponding result with error message and exit code
+     */
+    private fun exitWithError(errMsg: String): ExecutionResult {
```

Comment:
Pls rename to something like `returnWithError`, I thought that you are terminating the whole process by this method's name :)

Reply:
Agree, it will be more appropriate. Renamed